dfe-analytical-services/rstudio-connect-guidance
# Added VSTS Release Agent Setup Chapter

**Overview**

1. Converted the presentation you provided into markdown and added as chapter in Annexe with a few tweaks. 
2. Updated travis to run on r 3.4.2 as it was just latest version before which is 3.5.0 and i've had problems with existing code. 

**QA**

1. Download and check everything renders properly
2. Read through and see if there are any changes you disagree with
3. Maybe have a think about spacing the images out a bit more. Have you got any css that does this?
